patches/minecraft/net/minecraft/world/storage/loot/LootTableManager.java.patch

```diff
@@ -0,0 +1,11 @@
+--- ../src-base/minecraft/net/minecraft/world/storage/loot/LootTableManager.java
++++ ../src-work/minecraft/net/minecraft/world/storage/loot/LootTableManager.java
+@@ -64,7 +64,7 @@
+             else
+             {
+                 LootTable loottable = this.func_186517_b(p_load_1_);
+-
++                if(loottable == null) loottable = net.minecraftforge.common.loot.LootRegistry.loadModLootTable(p_load_1_, field_186526_b);
```

Comment:
We may want to add something here that will tag the loot table as a 'world override' so that any add/remove methods get nooped so we garentee that world jsons don't get tampered with by mods. {As that's the point of world ones, to be full and exact by the map maker not the modders}